substrate/runtime/staking/src/lib.rs

```diff
@@ -57,7 +57,8 @@ use codec::{Input, Slicable};
 use runtime_support::{StorageValue, StorageMap, Parameter};
 use runtime_support::dispatch::Result;
 use primitives::traits::{Zero, One, Bounded, RefInto, SimpleArithmetic, Executable, MakePayment,
-	As, Lookup, Hashing as HashingT, MaybeSerializeDebug, Member};
+	As, Lookup, Hashing as HashingT, Member};
+use address::Address as RawAddress;
```

Comment:
why?

Reply:
it doesn't build if you use the type alias in the macro. just a workaround